distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -196,5 +197,3 @@ log4j2.appender.artemis.policies.size.type = SizeBasedTriggeringPolicy
 log4j2.appender.artemis.policies.size.size = 20MB
 log4j2.appender.artemis.strategy.type = DefaultRolloverStrategy
 log4j2.appender.artemis.strategy.max = 10
-
-# Setting custom configurations using log:set from the console will append those configurations below this line
```

Comment:
Nice change! I confirmed that custom configurations using `log:set` from the console are appended after the last line of configuration in this file, not appended to the end of the file.